adafruit_74hc595.py

```diff
@@ -44,6 +44,8 @@ def __init__(self, pin_number, shift_register_74hc595):
         ShiftRegister74HC595 instance.
         """
         self._pin = pin_number
+        self._byte_pos = int(self._pin / 8)
```

Comment:
This will do truncated division:
```suggestion
        self._byte_pos = self._pin // 8
```